src/integrationTest/java/org/opensearch/security/InternalAuditLogTest.java

```diff
@@ -48,13 +48,24 @@ public class InternalAuditLogTest {
         .build();
 
     @Test
-    public void testAuditLogShouldBeGreenInSingleNodeCluster() throws IOException {
+    public void testAuditLogShouldBeGreenInSingleNodeCluster() throws InterruptedException {
         try (TestRestClient client = cluster.getRestClient(USER_ADMIN)) {
             client.get(""); // demo request for insuring audit-log index is created beforehand
-            TestRestClient.HttpResponse indicesResponse = client.get("_cat/indices");
-
-            assertThat(indicesResponse.getBody(), containsString("security-auditlog"));
-            assertThat(indicesResponse.getBody(), containsString("green"));
+            int retriesLeft = 5;
```

Comment:
Are you seeing flakiness in this test? I'm ok w/ putting the retry logic in, but the tests in the `integrationTest` sourceSet are already run with retries. This test is new and ideally should not have introduced flakiness. Is there a race condition between when the audit log index is created and when its available in the output of `_cat/indices`?

Reply:
Yes. It failed on my CI run, and when I tested on my own computer it always failed without small sleep just before client.get("_cat/indices"). To be honest, I did not examine it any further.

Reply:
We'll address the flakiness in another PR to wait for the cluster to be green before calling `_cat/indices`. Thanks for putting the basic retry logic in here.